mozilla/donate-wagtail
# Backend postcode validation

Closes #367 

Review App can be found at: https://donate-wagta-backend-po-oqi62q.herokuapp.com/en-US/

**Steps to test country with post code:** 

1.) Visit [Donate Site Review App](https://donate-wagta-backend-po-oqi62q.herokuapp.com/en-US/)
2.) Select Canada/US/Other country that requires post code
3.) Fill out information for all fields under "Your Information" except ZIP/Postal Code
4.) Fill out card information: 
(Example Card Info, please feel free to use these):
Credit Card: 4111 1111 1111 1111
Exp: 11/23
CVV: 123

5.) Submit the Form
6.) The ZIP code input should now be displaying an error state (red border/red error message underneath)

**Steps to test country without post code:** 

1.) Visit [Donate Site Review App](https://donate-wagta-backend-po-oqi62q.herokuapp.com/en-US/)
2.) Select Tonga/Other country that does not require post code
3.) The ZIP field should now dissapear, just leaving the city field.
3.) Fill out information for all fields under "Your Information" 
4.) Fill out card information: 
(Example Card Info, please feel free to use these):
Credit Card: 4111 1111 1111 1111
Exp: 11/23
CVV: 123

5.) Submit the Form
6.) You should see the thank you page

And thats it! Thank you for taking the time to test
 

## Checklist

**Changes in Models:**
